sources/packages/api/src/database/config/ormconfig.ts

```diff
@@ -19,9 +21,10 @@ module.exports = {
   schema: process.env.DB_SCHEMA || "sims",
   synchronize: false,
   migrations,
-  cli: {
-    migrationsDir: "src/database/migrations",
```

Comment:
Why did we remove this config? how will the typeorm know which directory to scan?